cmd/tailscale/cli/configure-kube.go

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+//go:build !ts_omit_kube
```

Comment:
Let's also add `ts_small` too. I filed https://github.com/tailscale/tailscale/issues/7230 

So this is then

`!ts_omit_kube && !ts_small`

Reply:
I'll add another commit that does it so that it's a single change that spans the repo as opposed to being introduced in one commit in one place.

Reply:
Done. PTAL

Reply:
Realized that we already have `./build_dist.sh --extra-small` so not going to add `ts_small` as a tag and instead make that the supported way.